src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -3259,20 +3259,17 @@ internal static void SetSessionStateDrive(ExecutionContext context, bool setLoca
 
```

Comment:
@iSazonov, your last commit had 84 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Clean up open Runspaces when exit powershell process.PowerShell process should not freeze at exit
```powershell
No such file or directory
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Api/BasicEngine.Tests.ps1: line 52
52:         $process = Start-Process pwsh -ArgumentList $command -PassThru -RedirectStandardOutput $outputFile

```
Enable-ExperimentalFeature and Disable-ExperimentalFeature tests.Disable-ExperimentalFeature will disable Experimental Feature for scope: AllUsers
```powershell
Program 'pwsh' failed to run: No such file or directoryAt /home/vsts/work/1/s/test/powershell/engine/ExperimentalFeature/EnableDisable-ExperimentalFeature.Tests.ps1:80 char:20
+ ...  $feature = pwsh -noprofile -output xml -command Get-ExperimentalFeat ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/ExperimentalFeature/EnableDisable-ExperimentalFeature.Tests.ps1: line 80
80:         $feature = pwsh -noprofile -output xml -command Get-ExperimentalFeature ExpTest.FeatureOne

```
Basic SysLog tests on Linux.Verifies basic logging with no customizations
```powershell
Program 'pwsh' failed to run: No such file or directoryAt /home/vsts/work/1/s/test/powershell/Host/Logging.Tests.ps1:173 char:9
+         & $powershell -NoProfile -SettingsFile $configFile -Command ' ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Logging.Tests.ps1: line 173
173:         & $powershell -NoProfile -SettingsFile $configFile -Command '$env:PSModulePath | out-null'

```
Enable-ExperimentalFeature and Disable-ExperimentalFeature tests.Enable-ExperimentalFeature will enable Experimental Feature for scope: AllUsers
```powershell
Program 'pwsh' failed to run: No such file or directoryAt /home/vsts/work/1/s/test/powershell/engine/ExperimentalFeature/EnableDisable-ExperimentalFeature.Tests.ps1:61 char:20
+ ...  $feature = pwsh -noprofile -output xml -command Get-ExperimentalFeat ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/ExperimentalFeature/EnableDisable-ExperimentalFeature.Tests.ps1: line 61
61:         $feature = pwsh -noprofile -output xml -command Get-ExperimentalFeature ExpTest.FeatureOne

```
Basic SysLog tests on Linux.Verifies scriptblock logging with null character
```powershell
Program 'pwsh' failed to run: No such file or directoryAt /home/vsts/work/1/s/test/powershell/Host/Logging.Tests.ps1:229 char:17
+ ...     $null = & $powershell -NoProfile -SettingsFile $configFile -Comma ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Logging.Tests.ps1: line 229
229:         $null = & $powershell -NoProfile -SettingsFile $configFile -Command $testScriptPath

```